sktree/tree/_neighbors.py

```diff
@@ -64,3 +64,7 @@ def compute_similarity_matrix(self, X):
             The similarity matrix among the samples.
         """
         return compute_forest_similarity_matrix(self, X)
+
+    def _more_tags(self):
+        # XXX: no scikit-tree estimators support NaNs as of now
```

Comment:
Scikit-tree will depend on each version of scikit-learn successively very tightly for now, so we'll consider the next release of v0.8 to be tied to 1.4.x. Is that okay?

Reply:
Right now, when there is no missing value support, then it would be okay to attach the same "disable all"-type`_more_tags` method to the tree ABC, so that it affects all tree and tree ensemble estimators.

The exact SkTree <-> SkLearn version matching is not critical until SkTree really starts enabling missing values.